lib/tasks/say.js

```diff
@@ -21,15 +21,26 @@ class TaskSay extends Task {
     const {updateSpeechCredentialLastUsed} = require('../utils/db-utils')(this.logger, srf);
     const {writeAlerts, AlertType, stats} = srf.locals;
     const {synthAudio} = srf.locals.dbHelpers;
-    const hasVerbLevelTts = this.synthesizer.vendor && this.synthesizer.vendor !== 'default';
-    const vendor = hasVerbLevelTts ? this.synthesizer.vendor : cs.speechSynthesisVendor ;
-    const language = hasVerbLevelTts ? this.synthesizer.language : cs.speechSynthesisLanguage ;
-    const voice = hasVerbLevelTts ? this.synthesizer.voice : cs.speechSynthesisVoice ;
+    const vendor = this.synthesizer.vendor && this.synthesizer.vendor !== 'default'
+      ? this.synthesizer.vendor
+      : cs.speechSynthesisVendor;
+    const language = this.synthesizer.language && this.synthesizer.language !== 'default'
+      ? this.synthesizer.language
+      : cs.speechSynthesisLanguage ;
+    const voice =  this.synthesizer.voice && this.synthesizer.voice !== 'default'
+      ? this.synthesizer.voice
+      : cs.speechSynthesisVoice;
     const engine = this.synthesizer.engine || 'standard';
     const salt = cs.callSid;
     const credentials = cs.getSpeechCredentials(vendor, 'tts');
 
-    this.logger.info({language, voice}, `Task:say - using vendor: ${vendor}`);
+    this.logger.info({language,
```

Comment:
this looks like it was useful for debugging purposes, but its a lot of logging that doesn't seem to add much value at this point.